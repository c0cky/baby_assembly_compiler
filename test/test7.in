3 10 9 12 11 13 12 0
