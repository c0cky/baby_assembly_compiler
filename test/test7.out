7
70
10
